Framework/include/QualityControl/CheckRunner.h

```diff
@@ -193,9 +194,12 @@ class CheckRunner : public framework::Task
 
   // monitoring
   std::shared_ptr<o2::monitoring::Monitoring> mCollector;
-  std::chrono::system_clock::time_point startFirstObject;
-  std::chrono::system_clock::time_point endLastObject;
+  //  std::chrono::system_clock::time_point startFirstObject;
+  //  std::chrono::system_clock::time_point endLastObject;
   int mTotalNumberHistosReceived;
```

Comment:
Could you take an opportunity to change it to `mTotalNumberMOReceived` for example? Histos are a bit misleading.

Reply:
totally agree